lib/ansible/plugins/inventory/tower.py

```diff
@@ -100,8 +104,9 @@
 from ansible.module_utils import six
 from ansible.module_utils.urls import Request, urllib_error, ConnectionError, socket, httplib
 from ansible.module_utils._text import to_native
```

Comment:
```suggestion
from ansible.module_utils._text import to_native, to_text
```

(This goes along with my comment about using to_text instead of to_native in controller code.